### PR TITLE
Releasing activity for Beaker 4.42.0

### PR DESCRIPTION
**Pull request by span786**
Releasing activity for Beaker 4.42.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [4.41.3](https://github.com/voxpupuli/beaker/tree/4.41.3) (2024-03-27)
+
+[Full Changelog](https://github.com/voxpupuli/beaker/compare/4.41.2...4.41.3)
+
+**Implemented enhancements:**
+
+- \[PE-37978\] Add 'amazon' to repo_filename method with tests [\#1861](https://github.com/voxpupuli/beaker/pull/1861) ([span786](https://github.com/span786))
```
Comment (bastelfreak) [inline CHANGELOG.md:9]: There's another enhancement that should be listed here: https://github.com/voxpupuli/beaker/compare/4.41.2...4.x

File: lib/beaker/version.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 module Beaker
   module Version
-    STRING = '4.41.2'
+    STRING = '4.41.3'
```
Comment (bastelfreak) [inline lib/beaker/version.rb:3]: this this release contains an enhancement it has to be a minor release. Please make it 4.42.0

File: CHANGELOG.md (hunk 1 above)
Comment (span786) [reply CHANGELOG.md:9]: Updated the changelog with all the enhancements

File: lib/beaker/version.rb (hunk 2 above)
Comment (span786) [reply lib/beaker/version.rb]: Updated the release version to 4.42.0

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**